src/cardboard-vr-display.js

```diff
@@ -31,6 +31,7 @@ var Eye = {
  * VRDisplay based on mobile device parameters and DeviceMotion APIs.
  */
 function CardboardVRDisplay() {
+  this.isPolyfilled = true;
```

Comment:
I just realised I can remove these since these displays inherit from `VRDisplay`, which also sets the prop